frontend/src/questions/components/LabelIconPicker.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component, PropTypes } from "react";
+
+import S from "./LabelIconPicker.css";
```

Comment:
I figured if it's an application-wide convention[1] it's probably not a big deal. It's used so much when using CSS modules and every character adds up.

If you feel strongly I can switch though.

[1] To be explicit, the convention I was going for was `S` always refers to the stylesheet corresponding to the same filename as the `.jsx` but with the `.css` extension (or the slightly relaxed version I'm currently using where the `.css` file can be shared among several related components, e.x. `Question.css` for `QuestionItem.jsx` and `QuestionList.jsx`)